Resources/Prototypes/Entities/Constructible/Walls/walls.yml

```diff
@@ -554,3 +554,10 @@
   - type: IconSmooth
     key: walls
     base: wood
+  - type: Flammable
+  - type: AtmosExposed
+  - type: Appearance
+    visuals:
+      - type: FireVisualizer
+        sprite: Mobs/Effects/onfire.rsi
+        normalState: Monkey_burning
```

Comment:
are these slime ppl changes
![image](https://user-images.githubusercontent.com/16618384/108687312-de94c280-74f6-11eb-8e2c-b7adee664439.png)

Reply:
also wat. `monkey_burning`
whats happening here

Reply:
oops, this is a lil' test I was doing for a different thing! I'll fix it..... tomorrow